lib/Traits/TraitsClasses/ItemPropertiesTraits.ts

```diff
@@ -0,0 +1,69 @@
+import { JsonObject } from "../../Core/Json";
+import anyTrait from "../Decorators/anyTrait";
+import objectArrayTrait from "../Decorators/objectArrayTrait";
+import primitiveTrait from "../Decorators/primitiveTrait";
+import ModelTraits from "../ModelTraits";
+
+export class ItemPropertiesByTypeTraits extends ModelTraits {
+  @primitiveTrait({
+    name: "Type of model",
+    description:
+      "The type of model to apply `itemProperties` to. This must be defined.",
+    type: "string"
+  })
+  type?: string;
+
+  @anyTrait({
+    name: "Item Properties",
+    description:
+      "Sets traits on group member items of specified `type`. This applies to all nested group members (eg members in sub-groups)"
+  })
+  itemProperties?: JsonObject;
+}
+
+export class ItemPropertiesByIdTraits extends ModelTraits {
+  @primitiveTrait({
+    name: "ID of model",
+    description:
+      "The ID of model to apply `itemProperties` to. This must be defined.",
+    type: "string"
+  })
+  id?: string;
```

Comment:
Should this be an array (`ids`)? I think many times we'll want to apply to multiple items.

Reply:
Good idea! done